<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCameraPilotageClass</class>
 <widget class="QMainWindow" name="QtCameraPilotageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtCameraPilotage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btncamTiltDown">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string> ↓ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPowerOff">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>290</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eteindre Camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncamPanLeft">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncamScan">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Balayage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoomWideStandard">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dezoomer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btncamPanRight">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZoomTeleStandard">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoomer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPowerOn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Allumer camera</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btncamTiltUp">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectPort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtCameraPilotage.qrc"/>
 </resources>
 <connections/>
</ui>
